netbout-web/pom.xml

```diff
@@ -326,13 +326,15 @@
                             <npmVersion>2.11.3</npmVersion>
                         </configuration>
                     </execution>
+                    <!-- @todo #914 investigate error messages of the following
+                         command and change package.json accordingly -->
```

Comment:
@jhyle also, the second line of the puzzle should be indented one line more compared to the previous lines